scripts/gen_argspecs.py

```
@@ -0,0 +1,210 @@
+                    break
+                if stripped.startswith("# @var"):
+                    # A new @var starts (typically an :example: sibling) —
+                    # description ends here, leave i pointing at that line.
+                    break
+                if not stripped.startswith("#"):
+                    # Docblock ended without @end — leave i pointing at
+                    # the value line and let the value-line branch handle it.
+                    break
+                desc_lines.append(lines[i].lstrip("# ").rstrip())
+                i += 1
+            pending_desc = " ".join(l for l in desc_lines if l).strip()
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)